packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -192,7 +192,16 @@ class AndroidDevice extends Device {
       // http://developer.android.com/ndk/guides/abis.html (x86, armeabi-v7a, ...)
       switch (await _getProperty('ro.product.cpu.abi')) {
         case 'arm64-v8a':
-          _platform = TargetPlatform.android_arm64;
+          // Perform additional verification for 64 bit ABI. Some devices,
+          // like the Kindle Fire 8, misreport the abilist. We might not
+          // be able to retrieve this property, in which case we fall back
+          // to assuming 64 bit.
+          final String gabilist = await _getProperty('ro.product.cpu.abilist');
```

Comment:
What is gabilist with the g?

Reply:
I don't actually know, I'll change it to abilist